subsys/bluetooth/controller/ll_sw/ull_iso_types.h

```diff
@@ -9,11 +9,18 @@
 #define LL_BIS_ADV_HANDLE_BASE BT_CTLR_ADV_ISO_STREAM_HANDLE_BASE
 #define LL_BIS_ADV_IDX_FROM_HANDLE(conn_handle) \
 	((conn_handle) - (LL_BIS_ADV_HANDLE_BASE))
+/* Conditional compile to prevent coverity issue CWE570, comparison of unsigned int to 0 */
+#if (LL_BIS_ADV_HANDLE_BASE > 0)
 #define IS_ADV_ISO_HANDLE(conn_handle) \
 	(((conn_handle) >= (LL_BIS_ADV_HANDLE_BASE)) && \
 	 ((conn_handle) <= ((LL_BIS_ADV_HANDLE_BASE) + \
 			    (BT_CTLR_ADV_ISO_STREAM_MAX) - 1U)))
 #else
+#define IS_ADV_ISO_HANDLE(conn_handle) \
+	((conn_handle) <= ((LL_BIS_ADV_HANDLE_BASE) + \
```

Comment:
so LL_BIS_ADV_HANDLE_BASE is not 'needed' here?

Reply:
Correct, but I would like to keep it this way so that the two formulas are as similar as possible